packages/replayio/src/commands/upload.ts

```diff
@@ -0,0 +1,62 @@
+import chalk from "chalk";
+import { requireAuthentication } from "../utils/authentication/requireAuthentication";
+import { registerCommand } from "../utils/commander";
+import { confirm } from "../utils/confirm";
+import { exitProcess } from "../utils/exitProcess";
+import { getRecordings } from "../utils/recordings/getRecordings";
+import { processUploadedRecordings } from "../utils/recordings/processUploadedRecordings";
+import { selectRecordings } from "../utils/recordings/selectRecordings";
+import { LocalRecording } from "../utils/recordings/types";
+import { uploadRecordings } from "../utils/recordings/uploadRecordings";
+
+registerCommand("upload")
+  .argument("[ids]", `Recording ids ${chalk.dim("(comma-separated)")}`)
+  .option("-a, --all", "Upload all recordings")
+  .option("-p, --process", "Process uploaded recording(s)")
+  .description("Upload recording(s)")
+  .action(upload);
+
+async function upload(
+  ids: string[] | undefined,
+  {
+    all = false,
+    process: shouldProcess,
+  }: {
+    all?: boolean;
+    process?: boolean;
+  } = {}
+) {
+  await requireAuthentication(false);
+
+  const recordings = await getRecordings();
+
+  let selectedRecordings: LocalRecording[] = [];
+  if (ids && ids.length > 0) {
+    selectedRecordings = recordings.filter(recording => ids.includes(recording.id));
```

Comment:
It would be nice to warn the user that one of the specified IDs couldn't be found